Emby.Naming/Video/CleanStringParser.cs

```diff
@@ -19,7 +19,7 @@ public CleanStringResult Clean(string name, IEnumerable<Regex> expressions)
                 if (!string.IsNullOrEmpty(result.Name))
                 {
                     name = result.Name;
-                    hasChanged = hasChanged || result.HasChanged;
+                    hasChanged |= result.HasChanged;
```

Comment:
Why change it to bitwise OR?

Reply:
As you said, It doesn't matter here, and it just looks clear imo.

Reply:
I still feel we should really keep using semantically correct operators. Otherwise it starts to look too much like C/C++ hacks.

Reply:
@EraYaN I don't think it's out of place here. iirc VS would even recommend this.

Reply:
Mmm but it assumes an implementation detail that true = 0x1 and false is 0x0. (While bools might even be arbitrary values or objects). I mean sure in this context and for this language it might even be strictly defined, it just "feels" wrong if that makes sense.

Reply:
FYI this is done in multiple places in the current code base (some by me :p )

Reply:
VS doesn't care. I prefer `||`